src/bin/nydusd/main.rs

```diff
@@ -214,7 +217,14 @@ fn prepare_commandline_options() -> Command {
             Arg::new("config")
                 .long("config")
                 .short('C')
-                .help("Path to the Nydus daemon configuration file")
+                .help("Path to get the Nydus daemon configuration")
+                .required(false)
+                .global(true),
+        )
+        .arg(
+            Arg::new("config-source")
```

Comment:
Please add e2e test and update doc for new option

Reply:
I will add e2e test and update doc for new option, and there are examples of e2e tests that I can refer to?